APE0.rst

```diff
@@ -0,0 +1,360 @@
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than four weeks after the motion is seconded. Removal requires approval
+by two-thirds of all active Voting Members at the time the motion is made. The
+motion, second, and vote will be held in private. Removal under this provision
+will be reflected by updating the list of Voting Members.
+
+It may be necessary for the Ombudsperson and the Coordination Committee to
+remove a Voting Member for violations of the Code of Conduct. In this case, the
+Coordination Committee and Ombudsperson will work together to make this
+decision.
+
+Approving and Modifying This Charter
+------------------------------------
+This document was submitted following the process in APE1_, and the normal APE
+acceptance procedures will be followed. The Coordination Committee at the time
+of submitting this APE are all co-authors and therefore will not override any
+consensus of the community on accepting the final version.
+
+Changes to this Charter after it has been accepted should follow the
+modification process in APE1_, with the exception that the final approval of the
+modification requires approval by a two-thirds vote of the Voting Members rather
+than approval by the Coordination Committee.
+
+Attribution and Acknowledgments
+-------------------------------
+The format and some of the structures outlined in this document are heavily
+inspired by the Python Language Governance structure
+(`PEP13 <https://www.python.org/dev/peps/pep-0013/>`_), the YT Project's Team
+Infrastructure
+(`YTEP 1776 <https://ytep.readthedocs.io/en/latest/YTEPs/YTEP-1776.html>`_), and
+earlier less-formal descriptions of the Astropy governance.
+
+.. _APE1: https://github.com/astropy/astropy-APEs/blob/master/APE1.rst
+
+
+Branches and Pull Requests
+==========================
+N/A
+
+Implementation
+==============
+This Charter enters into force upon this APE being accepted (see the last
+section of the description). At that time the ``GOVERNANCE.md`` file in the
+astropy repo should be updated to point to this document.
+
+Backward Compatibility
+======================
+This Charter supersedes previous un-codified governance understandings, but does
+not serve to invalidate the APE process or any other processes or policies that
+pre-date it and do not conflict.
+
+Alternatives
+============
+The Astropy Governance Working Group discussed a wide range of alternatives on
+both the broad scope of Project governance and details of this Charter. It is
+not practical to summarize that in the text of this APE, but the Working Group's
+running notes provide an excellent starting point for this discussion.
```

Comment:
Link?

Reply:
To preserve history, the doc needs to be locked first before linking.

Reply:
It's currently only open for view via link, but WG members can edit it.  Is that what you meant @pllim, or do you think we should also lock it from the WG members?

Reply:
Hmm... I was thinking about permanent, but I'll leave it to your discretion.

Reply:
Oh but for *immediate* purposes if others want to look, the link is here: https://docs.google.com/document/d/1XsJCQDm1EBWm2w2yDbohWw2HhMAO_-YhnOMhBsNV82I/edit?usp=sharing

Reply:
I added this link to the document, since I don't think we plan to publish it under some other platform